plivo/utils.go

```diff
@@ -56,14 +56,19 @@ func Headers(headers map[string]string) string {
 	return headersWithSep(headers, "=", ",", true)
 }
 
-func ComputeSignature(authToken, uri string, params map[string]string) string {
-	originalString := fmt.Sprintf("%s%s", uri, headersWithSep(params, "", "", false))
-	logrus.Infof("originalString: %s\n", originalString)
-	mac := hmac.New(sha1.New, []byte(authToken))
+func ComputeSignature(authToken, uri string, nonce string) string {
+	parsedUrl, err := url.Parse(uri)
+	if err != nil {
+		return "xxxxx"
```

Comment:
This code isn't following the coding style in comparison with the rest of the project.